bwpy/mea_viewer.py

```diff
@@ -0,0 +1,77 @@
+from . import signal
+import abc
+import plotly.graph_objects as go
+import numpy as np
+from scipy.stats import norm
+
+
+class Viewer(abc.ABC):
+    @abc.abstractmethod
+    def build_view(self, slice, bin_size, view_method):
+        pass
+
+
+class MEAViewer(Viewer):
+    colorscale = [[0, "#ebe834"], [1.0, "#eb4034"]]
+    min_value = 0
+    max_value = 0
```

Comment:
Can be removed since you fit them now. If the min value is always 0, just use 0 wherever you use it, that's usually clear enough and can be optimized away by CPython (probably?). Class variable lookups are the slowest since they are the last in the attribute lookup chain (something like 35ns in the official Python benchmarks, instead of just being inlined in the bytecode for a constant).